src/test/java/se/sundsvall/webmessagecollector/service/scheduler/MessageCacheServiceTest.java

```diff
@@ -60,7 +61,7 @@ class MessageCacheServiceTest {
 		""";
 
 	@Mock
-	private OpenEClient openEClientMock;
+	private OpenEIntegration openEClientMock;
```

Comment:
Think you missed refactoring the variable name.

Reply:
I think you might be right